packages/plugin-rsc/examples/react-router/cf/vite.config.ts

```diff
@@ -43,12 +43,12 @@ export default defineConfig({
     },
     ssr: {
       optimizeDeps: {
-        exclude: ['react-router'],
+        exclude: ['react-router', 'react', 'react-dom'],
       },
     },
     rsc: {
       optimizeDeps: {
-        exclude: ['react-router'],
+        exclude: ['react-router', 'react', 'react-dom'],
```

Comment:
This is unfortunate, but necessary anyways since react 3rd party libraries with potential `"use client"` cannot be optimized.